### PR TITLE
Document Widevine process

### PR DESCRIPTION
**Pull request by JohanLorenzo**

### CODE REVIEW

File: docs/misc-operations/widevine-updates.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+If the API level (or higher) is bumped, please check with the Media team what Firefox is able to run this Widevine.
+
+:warning: There are 2 types of patches the Media team makes:
+
+1. The actual patch to make Firefox compatible with the new API (for example: [bug 1420836](https://bugzilla.mozilla.org/show_bug.cgi?id=1420836)). This kind of patch must happen **before** the balrog rule is set.
+1. Another patch that updates the fallback downloader (like [bug 1479579](https://bugzilla.mozilla.org/show_bug.cgi?id=1479579)). This kind of patch can be landed **after** the balrog rule is set
+
+### Create the blob
+
+// TODO
```
Comment (JohanLorenzo) [inline docs/misc-operations/widevine-updates.md]: @lundjordan Would you have more info on this part? I haven't tried to set up a blob, yet.

File: docs/misc-operations/widevine-updates.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+### Ensure what Firefox version is able to run the new version
+
+Take a look at widevine version numbers. For instance: 1.4.9.1088.
+
+| Digit | Name         | Notes                                                             |
+|-------|--------------|-------------------------------------------------------------------|
+| 1     | ?            |                                                                   |
+| 4     | ?            |                                                                   |
+| 9     | API level    | If this number changes, some Firefox internals must be changed    |
```
Comment (escapewindow) [inline docs/misc-operations/widevine-updates.md]: Awesome, I didn't know this :)

**Comment by MihaiTabara** [comment]
I think it's worth mentioning this in the https://github.com/mozilla-releng/releasewarrior-2.0/blob/master/docs/CHANGELOG.md as well 👍

File: docs/misc-operations/widevine-updates.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+| Digit | Name         | Notes                                                             |
+|-------|--------------|-------------------------------------------------------------------|
+| 1     | Major        | It has always been 1, so far                                      |
+| 4     | Module       | Significant changes must happen on the Firefox side               |
+| 9     | Interface    | If this number changes, some Firefox internals must be changed    |
+| 1088  | Revision     | This number keeps increasing even if the other numbers got bumped |
+
+If the API level (or higher) is bumped, please check with the Media team what Firefox is able to run this Widevine.
+
+#### New schema (> 1.4.9.x)
```
Comment (JohanLorenzo) [inline docs/misc-operations/widevine-updates.md]: Thank you @SingingTree for filling the gaps and warning us about the new schema coming up!

File: docs/misc-operations/widevine-updates.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+# Widevine updates
+
+Widevine is a system addon allowing Firefox user to read DRM'd content (like Netflix). We provide updates via Balrog.  
+
+## When
+
+The request comes from the media team. They usually file a bug like this one.
```
Comment (tomprince) [inline docs/misc-operations/widevine-updates.md]: Is `this one` meant to be a link?

File: docs/CHANGELOG.md [hunk 5]
```diff
@@ -8,6 +8,7 @@ _**As releaseduty squirrels are the ones with the best context when it comes to
 
 ### Added
 - added partner off-cycle support + docs
+- widevine docs
```
Comment (lundjordan) [inline docs/CHANGELOG.md]: Thanks for remembering this!

File: docs/misc-operations/widevine-updates.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+|-------|--------------|-------------------------------------------------------------------|
+|       | Major        | See old schema                                                    |
+|       | Interface    | See old schema                                                    |
+|       | Revision     | See old schema                                                    |
+|       | ?            | TDB                                                               |
+
+
+### Create the blob
+
+// TODO
```
Comment (lundjordan) [inline docs/misc-operations/widevine-updates.md]: I'll fill this in.

File: docs/misc-operations/widevine-updates.md (hunk 4 above)
Comment (lundjordan) [reply docs/misc-operations/widevine-updates.md]: I'll update this

**Pull request merged**